src/ext/ca/wixca/dll/secureobj.cpp

```diff
@@ -586,6 +591,7 @@ extern "C" UINT __stdcall ExecSecureObjects(
     DWORD dwRevision = 0;
     LPWSTR pwzUser = NULL;
     DWORD dwPermissions = 0;
+    int isInheritable = 0;
```

Comment:
This is really a BOOL right?  If so, then you'd want `BOOL fInheritable` and if it really should be a numeric, then you'd want a DWORD so `DWORD dwInheritable`.

Actually, based on comment above, probably best to make this column the `Attributes` column and make Inheritable a bit flag in that column. Future proof ourselves, finally.